sensitivity.tex

```diff
@@ -12,18 +12,19 @@ \section{Sensitivity Analysis}
 
 \begin{table}[h]
     \centering
-    \begin{tabularx}{\textwidth}{lbb}
+    \caption {Two parameters were varied for the sensitivity analysis - the 
```

Comment:
I recommend just one sentence instead of two. "Two parameters were varied - the lifetime of French \glspl{LWR} and the breeding ratio of \glspl{ASTRID} - to analyze their impact on the reprocessing demand during the transition."